README.md

```diff
@@ -42,6 +42,7 @@ How to install protoc-gen-grpc-web: https://github.com/grpc/grpc-web#code-genera
 # generate proto messages and the service client stub classes
 npm run build:proto
 ```
+> The version of `protobuf` uses 3.4.0 or higher.
```

Comment:
The source of truth for protobuf message is in [yorkie](https://github.com/yorkie-team/yorkie/tree/main/api). So I added a description of the protobuf version to the yorkie project (https://github.com/yorkie-team/yorkie#developing-yorkie). I think it would be nice to just mention the source location of the protobuf message in yorkie-js-sdk.

```suggestion
> Primary "source of truth" location of protobuf message is in [yorkie](https://github.com/yorkie-team/yorkie/tree/main/api). We manage the messages in the repository.
```